modules/tinymce/src/plugins/table/main/ts/ui/UiUtils.ts

```diff
@@ -74,11 +74,19 @@ const generateColorSelector = (editor: Editor, colorList: Menu.ChoiceMenuItemSpe
   }
 }];
 
+const changeRowHeader = (editor: Editor) =>
+  () => {
```

Comment:
I'm not sure about this being on a new line, interested to hear others' thoughts.

Reply:
Yeah, normally we'd keep this on a single line.